ros_buildfarm/status_page_input.py

```diff
@@ -98,5 +98,12 @@ def get_rosdistro_info(dist, build_file):
             except InvalidPackage:
                 pass
 
+        # handle ignored packages
+        if pkg_name not in filtered_pkg_names:
+            ros_pkg.version = None
```

Comment:
A `None` version is what tells the status page generation to use grey boxes on the right side of the page.